cluster/sbin/snappy-nodes.sh

```diff
@@ -338,12 +346,44 @@ function getNumLeadsOnHost() {
   echo $numLeadsOnHost
 }
 
+function scpConf() {
```

Comment:
You can add a one line comment describing what the function does. Also, name can be more intuitive.

Reply:
Added comment